backend/repository/userrepo.go

```diff
@@ -69,27 +70,25 @@ func (repo UserRepository) GetUserTimeline(user_id uint) ([]model.RenderMessage,
 	return messages, nil
 }
 func (repo UserRepository) GetIsFollowing(who uint, whom uint) bool {
-	followed := false
-	repo.db.Raw("select 1 from follower where follower.who_id = ? and follower.whom_id = ?", who, whom).Scan(&followed)
-	return followed
+	var f model.Follower
+	// If first cannot find a value, this query will throw an error.
```

Comment:
I feel like these two comments add no value as they just describe the code, and clutter the codebase